stdlib/3/subprocess.pyi

```diff
@@ -199,6 +199,8 @@ class CalledProcessError(Exception):
     def __init__(self, returncode: int, cmd: str, output: Optional[str] = ...,
                  stderr: Optional[str] = ...) -> None: ...
 
+class TimeoutExpired(SubprocessError): ...
```

Comment:
You may have noticed an error in the tests for Python 3.2. That's because SubprocessError is only defined for Python >= 3.3. If you move this definition into the conditional block that defines SubprocessError (at line 187) the problem should be solved.